low-level/sensors/impl/generic/node_hw.py

```diff
@@ -252,6 +236,10 @@ def initialize(self, conf_reader, msgQlist, products):
         # Initialize internal message queues for this module
         super(NodeHWsensor, self).initialize_msgQ(msgQlist)
 
+        ipmi_client = Conf.get(SSPL_CONF, f"{NODEHWSENSOR}>{IPMI_CLIENT}",
+                               "ipmitool")
+        self.ipmi_tool = IpmiFactory().get_implementor(ipmi_client)
```

Comment:
Hi @cdeshmukh, agreed, since I am not aware of other tools, any other name is not popping in my mind, can you suggest please? should it be `ipmi_client` instead and variable above can be renamed as `ipmi_client_impl` ??